pkg/kubelet/kubelet_pods.go

```diff
@@ -487,6 +502,16 @@ func managedHostsFileContent(hostIPs []string, hostName, hostDomainName string,
 	return buffer.Bytes()
 }
 
+// isSupportIPv6 determines whether the host supports ipv6 by check the global ipv6 support sysctl.
+func isSupportIPv6() bool {
```

Comment:
Just nitpicking the English a little bit, maybe "ipv6Supported" or "isIPv6Supported" ?